tests/tests_user_config.yml

```diff
@@ -0,0 +1,66 @@
+---
+- name: Test a creation of configuration file in the user directory
+  hosts: all
+  vars:
+    username: tester123
+  tasks:
+    - name: Create a test group
```

Comment:
Does this really create just a group? I didn't have this impression from documentation.

Reply:
Indeed, the module here should be group instead of `user`. Let me fix that.